morl_baselines/multi_policy/multi_policy_moqlearning/mp_mo_q_learning.py

```diff
@@ -19,86 +28,76 @@ class MPMOQLearning(MOAgent):
     def __init__(
         self,
         env,
-        ref_point: np.ndarray,
```

Comment:
I see ref_point as an evaluation variable, like test_weights or eval_env, for instance. So in my head makes sense that it is an argument train instead of the class constructor. Another option would be to have the envs in mo-gymnasium to have ref_point as attribute? (e.g env.ref_point)

Reply:
Makes sense, let's put it next to `eval_env` then. I'm not sure about including these in MOMDPs for now... It is information needed by the algorithms, not by the MOMDPs (let's avoid mixing stuff up)